src/Facade/Schema.php

```diff
@@ -17,6 +17,17 @@
  */
 class Schema extends Facade
 {
+    /**
+     * Get a schema builder instance for a connection.
+     *
+     * @param  string  $name
+     * @return \Illuminate\Database\Schema\Builder
```

Comment:
missing return type? You could also use a use statement here

Reply:
This was a direct copy of Laravel :see_no_evil: , fixed it now.